schedule/yam/agama-from-usb.yaml

```diff
@@ -0,0 +1,12 @@
+---
+name: agama
+description: >
+  Perform interactive installation with agama.
+schedule:
+  - yam/agama/boot_agama
+  - yam/agama/patch_agama_tests
+  - yam/agama/agama
+  - yam/agama/boot_from_hd_after_agama
+  - installation/first_boot
```

Comment:
in the future I would like to create something like `boot_installed_system` that would process this task as a transparent workaround behind the curtains but will do the grub handling (that you miss here) and the first boot check, all in one module. I remember I told you to not to save the world in this ticket and narrow the scope to this single issue but on the other hand I see our yaml counter increasing, could you file a ticket with the information in this comment to act after this is merged?